Automattic/jetpack
# Classic Theme Helper:  featured content usage from module to package

## Proposed changes:

* Updated  `Classic Theme Helper` to use namespace and changed the main class of the package.
* Use the `Classic Theme Helper` package instead of the `Theme Tools` module.
* Deprecate existing class and functions in the `Theme Tools `Module.


### Other information:
**TODO: (In a following PR)** Needs wpcom diff to move specific WPCOM functions to the featured content mu-plugin loading the feature on simple.


## Jetpack product discussion
pfwV0U-6Y-p2



## Does this pull request change what data or activity we track or use?



## Testing instructions:






Test for Simple, Atomic and Self-Hosted ( Make sure there are no related server logs and browser errors )

### Test Themes Without Support
* Select a Classic Theme that does not support Featured Content
* Go to the Customizer 
* Check that Featured Content Section is **not** present

### Test Themes With Support ([Video](https://wordpress.com/support/customizer/featured-content/#video-overview) that can be helpful)
* Select a Classic Theme that does support Featured Content (Dara Theme as an example)
* Go to the Customizer 
* Check that Featured Content Section is present
* Add a Tag of your own like `my_tag` under Tag Name and save the changes
* Add the same tag to a few posts
* Go to the frontend and check that you can see the "Featured posts"
